src/components/Autocomplete/Autocomplete.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import _ from 'lodash';
 import { lucidClassNames } from '../../util/style-helpers';
-import { createLucidComponentDefinition } from '../../util/component-definition';
+import { createClass } from '../../util/component-definition';
```

Comment:
this wasn't strictly necessary for this, but I had already done the work of changing them all to use `createClass` and it sets us up for some refactoring work on child component definition and `createLucidComponentDefinition`, so i left it